### DISCUSSION TITLE
Problema riconnessione HA

### DISCUSSION THREAD

**Discussion by mikaletto** [indomus/forum#2465] [HUB: Home Assistant]
Salve con il mio PC con hassio in VM sono connesso wi fi del modem,
ma ogni 3-4 giorni perde la connessione e non c'è modo di riprenderla se con il riavvio della VM o del PC.
Mi sapreste aiutare per risolvere questo problema? Grazie mille a tutti

**Comment by snakuzzo** [comment]
In base a questi pochi "indizi" difficile darti qualche suggerimento

**Reply by mikaletto** [reply]
Grazie per l'interesse, di quale info avresti bisogno per qualche consiglio?
